diff --git a/node_modules/@react-navigation/native/lib/module/UnhandledLinkingContext.js b/node_modules/@react-navigation/native/lib/module/UnhandledLinkingContext.js
index 431a5c8..f720c8d 100644
--- a/node_modules/@react-navigation/native/lib/module/UnhandledLinkingContext.js
+++ b/node_modules/@react-navigation/native/lib/module/UnhandledLinkingContext.js
@@ -1,14 +1,9 @@
 "use strict";
 
 import * as React from 'react';
-const MISSING_CONTEXT_ERROR = "Couldn't find an UnhandledLinkingContext context.";
 export const UnhandledLinkingContext = /*#__PURE__*/React.createContext({
-  get lastUnhandledLink() {
-    throw new Error(MISSING_CONTEXT_ERROR);
-  },
-  get setLastUnhandledLink() {
-    throw new Error(MISSING_CONTEXT_ERROR);
-  }
+  lastUnhandledLink: undefined,
+  setLastUnhandledLink: () => {}
 });
 UnhandledLinkingContext.displayName = 'UnhandledLinkingContext';
 //# sourceMappingURL=UnhandledLinkingContext.js.map
diff --git a/node_modules/@react-navigation/native/src/UnhandledLinkingContext.tsx b/node_modules/@react-navigation/native/src/UnhandledLinkingContext.tsx
index 2e0f5bd..0caa64a 100644
--- a/node_modules/@react-navigation/native/src/UnhandledLinkingContext.tsx
+++ b/node_modules/@react-navigation/native/src/UnhandledLinkingContext.tsx
@@ -1,18 +1,11 @@
 import * as React from 'react';
 
-const MISSING_CONTEXT_ERROR =
-  "Couldn't find an UnhandledLinkingContext context.";
-
 export const UnhandledLinkingContext = React.createContext<{
   lastUnhandledLink: string | undefined;
   setLastUnhandledLink: (lastUnhandledUrl: string | undefined) => void;
 }>({
-  get lastUnhandledLink(): any {
-    throw new Error(MISSING_CONTEXT_ERROR);
-  },
-  get setLastUnhandledLink(): any {
-    throw new Error(MISSING_CONTEXT_ERROR);
-  },
+  lastUnhandledLink: undefined,
+  setLastUnhandledLink: () => {},
 });
 
 UnhandledLinkingContext.displayName = 'UnhandledLinkingContext';
